app/sidekiq/form526_failure_state_snapshot_job.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+# Log information about Form526Submission state to populate an admin facing Datadog dashboard
+class Form526FailureStateSnapshotJob
+  include Sidekiq::Job
+  sidekiq_options retry: false
+
+  attr_reader :start_date, :end_date
```

Comment:
Should the object be initialized with these dates as done in `Form526SubmissionProcessingReportJob`?

Reply:
good catch, need to remove this